CppCoreGuidelines.md

```diff
@@ -332,7 +332,7 @@ Also, suggest an improvement.
 ## <a name="SS-force"></a>In.force: Enforcement
 
 Rules with no enforcement are unmanageable for large code bases.
-Enforcement of all rules is possible only for a small weak set of rules or for a specific user community.
```

Comment:
Editors call: We think several of these are improvements, but we want to make sure that clarity comes first. This "weak" is an example where we don't have a consensus among the editors that this change is an improvement, could you remove this one please? Will note a few more below...

Reply:
reverted